micrometer-core/src/test/java/io/micrometer/core/instrument/binder/mongodb/MongoMetricsCommandListenerTest.java

```diff
@@ -149,4 +186,4 @@ void destroy() {
         }
     }
 
-}
+}
```

Comment:
Could you please add final newline (see `.editorconfig` file)?
(Most IDEs are supporting it OOB or through a plugin.)